### ISSUE TITLE
Can attack even without required stamina

### ISSUE THREAD

**Issue by TriangleAuthority** [Lknechtli/BetterCombatEvolved#22]
fix here: https://github.com/TriangleAuthority/BetterCombatEvolved/commit/e4e1af4dcf553c76593c708e36efede961db6e88

I should prob just contribute to this repo instead of linking commits...

**Comment by Chickenudoom** [comment]
I tried your script, works great!!!
Saw your comment regarding fistfights being untested - just tried that. Works the same as in regular combat, not enough stamina = no action performed. It's actually really enjoyable!!

**Comment by Lknechtli** [comment]
FYI- ESGO is also attacking the issue, but by slowing attack speed when you
run out of stamina.  Not sure I want to pull this in because it will
conflict with his changes.

**Comment by TriangleAuthority** [comment]
Ooh, I like that, too. Excited to see/poke around the new ESGO. Do you know if the dude has a git repo up somewhere?

**Comment by Lknechtli** [comment]
Nope, but I'm trying to convert him :) I can't wait for the next update, we
been chatting on steam pretty much the whole time and tossing ideas back
and forth

**Comment by TriangleAuthority** [comment]
Cool, would be nice at least so I can merge and make sure there aren't conflicts. Wouldn't mind adding some stuff as ESGO options either, even if it's just for faster playtesting.

**Comment by Lknechtli** [comment]
Might add a few options for this mod as well, from what he's shown me it's actually pretty easy

**Comment by BardBlue** [comment]
I'd like to see the option to use either a slow-attack or no-attack when stamina depleted. An option for slower dodges while stamina depleted too might be fun to play around for the especially masochistic  (and no rolls at all, though I can't remember if that's the case already). Or maybe you just don't any faster animation bonus from Kukassin's modified talents that give faster animations like Flat Footed.

I kind of wish Kukassin would just join Combat Evolved and apply his balance/gameplay changes here and leave the scaling aspect as a separate mod. Or you could merge your mods entirely and just add lots of options. I think it'd be easier for you guys to maintain compatibility and collaborate and it'd be easier for the user, too (less to update, no need to merge things.)

But I also understand if you want to keep things separate, too. Less potential for, shall we say, political strife.

**Comment by TriangleAuthority** [comment]
Wouldn't matter if he just threw a git repo up.

They're good as separate mods since there's tons of stuff in ESGO that's good for vanilla, too. Wouldn't want to be forced into BCEv.

Could add a BCEv preset though. That'd be nice and easy.

**Comment by TriangleAuthority** [comment]
So I'll probably make a pseudo fork of ESGO and add features there. If kukassin fixes bugs or adds something cool, I'll merge it. It'd save a ton of time on tweaking numbers, etc. I'll post a link when it happens, prob after he drops the next version.

He seems like kind of a difficult person... probably not to be relied upon in any case. It's w/e tho.
